Framework/TestWorkflows/test/test_MakeDPLObjects.cxx

```diff
@@ -49,6 +50,12 @@ void defineDataProcessing(std::vector<DataProcessorSpec> &specs) {
           v[999] = XYZ{1,2,3};
           ctx.allocator().snapshot(OutputSpec{"TST", "VECTOR"}, v);
           v[999] = XYZ{2,3,4};
+
+          // A snapshot for an std::vector of pointers to objects
+          std::vector<XYZ*> p;
+          for (auto & i : v) p.emplace_back(&i);
```

Comment:
hi, since you are in the end pushing back a pointer, using emplace_back() might be a bit confusing since it is meant to do in-place construction with arguments forwarded to the constructor. Now the compiler will do the correct thing here, but I had to squint for a while to figure out you are not actually constructing anything here.

Reply:
ok, changed now, I also added a comment